Kickstarter-iOS/DataSources/ProjectPamphletContentDataSource.swift

```diff
@@ -28,7 +28,9 @@ internal final class ProjectPamphletContentDataSource: ValueCellDataSource {
       inSection: Section.subpages.rawValue
     )
 
-    self.setRewardTitleArea(project: project)
+    if AppEnvironment.current.config?.features[Features.checkout.rawValue] != .some(true) {
```

Comment:
Hmm I think this might not be too trivial because it would require a change to how we're currently decoding the `Config` model. Given that we're looking to get rid of `Argo` decoding on this model, I'd be open to doing this refactor (including removing `Argo` decoding) as a separate item of work (maybe on investment day). What do you think?

Reply:
Thanks for the explanation...I wasn't sure if the reason was decoding or not...would be more than happy to defer this work to Argo deprecation 👍